ci-operator/step-registry/ipi/conf/aws/proxy/ipi-conf-aws-proxy-commands.sh

```diff
@@ -301,6 +301,7 @@ auth_param basic realm proxy
 acl authenticated proxy_auth REQUIRED
 http_access allow authenticated
 pid_filename /tmp/proxy-setup
+url_rewrite_host_header off
```

Comment:
http://www.squid-cache.org/Doc/config/url_rewrite_host_header/ seems to say that this setting is relevant when Squid redirects a request.  Is that accurate?  If so, what is the scenario in which Squid sends a redirect?  

Does this warrant a note in the documentation in case users are using Squid or another proxy that mangles headers by default?

Reply:
Looks like I misinterpreted some of the squid documentation. I think this setting is only relevant when squid is proxying a redirect request. 

I am digging further into squid documentation, forums, etc. and im starting to wonder if preserving the original host header through squid is even possible.